Readme.md

```diff
@@ -69,6 +69,22 @@ pip install git+git://github.com/DavHau/mach-nix@3.2.0
 ```shell
 nix-env -if https://github.com/DavHau/mach-nix/tarball/3.2.0 -A mach-nix
 ```
+or, if you prefer `nix-shell`:
+
++ if [Nix flakes is enabled](https://nixos.wiki/wiki/Flakes#:~:text=Installing%20flakes):
+
+  ```shell
+  nix shell github:DavHau/mach-nix
+  ```
+  
++ otherwise:
+  
+  ```shell
+  nix-shell -p '(callPackage (fetchTarball https://github.com/DavHau/mach-nix/tarball/3.2.0) {}).mach-nix'
+                                                                                      \---/
+                                                                                      release
+  ```
+    (To find out the latest release number, go to https://github.com/DavHau/mach-nix/releases/latest.)
```

Comment:
I think we don't need any comments about inserting releases etc. I will replace the release numbers in the readme on every release, so it will always be up to date.

Reply:
Removed the comments about releases. Thank you for your time!

Reply:
Wonderful! Thanks for your time as well.